clang/lib/StaticAnalyzer/Core/CheckerManager.cpp

```diff
@@ -365,13 +388,22 @@ namespace {
 
     void runChecker(CheckerManager::CheckBindFunc checkFn,
                     NodeBuilder &Bldr, ExplodedNode *Pred) {
+      llvm::TimeTraceScope TimeScope(checkerScopeName("Bind", checkFn.Checker));
       const ProgramPoint &L = PP.withTag(checkFn.Checker);
       CheckerContext C(Bldr, Eng, Pred, L);
 
       checkFn(Loc, Val, S, C);
     }
   };
 
+  llvm::TimeTraceMetadata getTimeTraceBindMetadata(SVal val) {
+    assert(llvm::timeTraceProfilerEnabled());
+    std::string name;
```

Comment:
Some leftover lowercase variable names here.

Reply:
Sorry for overlooking these
4cc3be9658c4 [NFC] Uppercase some variable names